File: abc/abc120/abc120_c.cs
// https://atcoder.jp/contests/abc120/submissions/16642816

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var S = Console.ReadLine();
        var l0 = S.Count(c => c == '0');
        var l1 = S.Count(c => c == '1');
        Console.WriteLine(2 * Math.Min(l0, l1));
    }
}

File: abc/abc120/abc120_c.fs
// https://atcoder.jp/contests/abc120/submissions/16643101

let S = stdin.ReadLine().ToCharArray()

S
|> Array.sumBy (fun c -> int c - int '0')
|> fun x -> 2 * min x (S.Length - x)
|> printfn "%d"
